ui/src/components/Nodes/NodesTable.vue

```diff
@@ -87,18 +120,35 @@ import { QueryParameters, UpdateModelFunction } from '@/types'
 import useQueryParameters from '@/composables/useQueryParams'
 import { FeatherInput } from '@featherds/input'
 import { FeatherSortHeader, SORT } from '@featherds/table'
-import { FeatherSortObject } from '@/types'
-import BreadCrumbs from '@/components/Layout/BreadCrumbs.vue'
-import { BreadCrumb } from '@/types'
+import { Category, FeatherSortObject, MonitoringLocation, Node } from '@/types'
+import { MainMenu } from '@/types/mainMenu'
+import { isNumber } from '@/lib/utils'
 
 const store = useStore()
 const sortStates: any = reactive({
   label: SORT.ASCENDING,
   location: SORT.NONE,
   foreignSource: SORT.NONE,
-  foreignId: SORT.NONE
+  foreignId: SORT.NONE,
+  sysContact: SORT.NONE,
+  sysLocation: SORT.NONE,
+  sysDescription: SORT.NONE,
+  flows: SORT.NONE
 })
 
+const currentSearch = ref('')
+const nodes = computed(() => store.state.nodesModule.nodes)
+const mainMenu = computed<MainMenu>(() => store.state.menuModule.mainMenu)
+const selectedCategories = computed<Category[]>(() => store.state.hierarchyModule.selectedCategories)
```

Comment:
Same comment here as above regarding using the store variables directly. Just wanted to make sure you knew. :) I'm also curious if you could say pull out just hierarchyModule into one variable, and then make several references in the template. All minor comments though, nothing to be done immediately. Overall this PR is looking great so far.